src/redux/missions/missions.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+
+const getUrl = 'https://api.spacexdata.com/v3/missions';
+
+const FETCHED_MISSION = 'FETCHED_MISSION';
+const JOIN_OR_LEAVE_MISSION = 'JOIN_OR_LEAVE_MISSION';
```

Comment:
There's no need to declare these constant action types (FETCHED_MISSION and JOIN_OR_LEAVE_MISSION) when using createAsyncThunk and createSlice. The advantage of Redux Toolkit is that it automatically generates action types. So for example in line 9 you can do:

'missions/fetchMissions', // String for the action type